config/config.go

```diff
@@ -47,6 +47,7 @@ type Config struct {
 	TurnAddress    string `default:":3478" required:"true" split_words:"true"`
```

Comment:
Just a little hint for testing, there is a hidden url parameter [`forceTurn=true`](https://github.com/screego/server/blob/ab187c14f3934d0e9fd3ddc2ff54ced7c93a1042/ui/src/useRoom.ts#L36) which forces the usage of turn, even if Stun would be enough. Example: https://app.screego.net/?room=sound-lavender-badger&forceTurn=true